src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -387,6 +387,46 @@ begin
   simp only [set.mem_to_finset, mem_neighbor_set],
 end
 
+/-- Given a subgraph `G'`and a set pairs, remove all of those pairs from the edge set
+of `G'` (if they were present). -/
+@[simps]
+def delete_edges (G' : G.subgraph) (s : set (sym2 V)) : G.subgraph :=
+{ verts := G'.verts,
+  adj := λ a b, G'.adj a b ∧ ¬ ⟦(a, b)⟧ ∈ s,
+  adj_sub := λ a b h', G'.adj_sub h'.1,
+  edge_vert := λ a b h', G'.edge_vert h'.1,
+  symm := λ a b h, by rwa [G'.adj_comm, sym2.eq_swap] }
+
+@[simp] lemma delete_edges_delete_edges (G' : G.subgraph) (s s' : set (sym2 V)) :
+  (G'.delete_edges s).delete_edges s' = G'.delete_edges (s ∪ s') :=
+begin
+  ext v,
+  { simp, },
+  { ext v w,
+    simp [and_assoc, not_or_distrib], },
+end
+
+@[simp] lemma delete_edges_empty_eq (G' : G.subgraph) : G'.delete_edges ∅ = G' :=
+by ext; simp
+
+lemma delete_edges_le (G' : G.subgraph) (s : set (sym2 V)) : G'.delete_edges s ≤ G' :=
+⟨by simp, by simp { contextual := tt }⟩
+
+lemma delete_edges_le_of_le (G' : G.subgraph) {s s' : set (sym2 V)} (h : s ⊆ s') :
+  G'.delete_edges s' ≤ G'.delete_edges s :=
+⟨by simp, λ v w, begin
+  simp only [delete_edges_adj, and_imp, true_and] { contextual := tt },
+  exact λ ha hn hs, hn (h hs),
+end⟩
+
+lemma coe_delete_edges_le (G' : G.subgraph) (s : set (sym2 V)) :
+  (G'.delete_edges s).coe ≤ (G'.coe : simple_graph G'.verts) :=
```

Comment:
Perhaps I'm missing something - but why isn't this an equality? Same for below

Reply:
These are meant to be the coercion versions of `delete_edges_le`.

(It might make sense defining `delete_edges` on `simple_graph` itself and then connecting things up with `(G'.delete_edges s).coe = G.coe.delete_edges s`.)

Reply:
Got it. Yeah that could make sense, up to you!